server/rematch/wsgi.py

```diff
@@ -1,11 +1,11 @@
 """
-WSGI config for rematch-server project.
+ WSGI config for rematch-server project.
 
-It exposes the WSGI callable as a module-level variable named ``application``.
+ It exposes the WSGI callable as a module-level variable named ``application``.
 
-For more information on this file, see
-https://docs.djangoproject.com/en/1.9/howto/deployment/wsgi/
-"""
+ For more information on this file, see
+ https://docs.djangoproject.com/en/1.9/howto/deployment/wsgi/
+ """
```

Comment:
Yes. You're right. It'll look better using comments instead of multiline text (as it is a comment).